packages/lexical-devtools/src/main.jsx

```diff
@@ -0,0 +1,12 @@
+import './index.css';
+
+import React from 'react';
+import ReactDOM from 'react-dom/client';
```

Comment:
TypeScript forces modules which don't explicitly have a `default` export to be imported in this way below: 
```suggestion
import * as React from 'react';
import * as ReactDOM from 'react-dom/client';
```

More info: https://www.typescriptlang.org/tsconfig#allowSyntheticDefaultImports